server/src/main/java/org/elasticsearch/common/util/concurrent/EsExecutors.java

```diff
@@ -387,4 +406,33 @@ private void reject(ThreadPoolExecutor executor, Runnable task) {
         }
     }
 
+    public static class TaskTrackingConfig {
+        private final boolean trackExecutionTime;
+        private final double EWMAAlpha;
```

Comment:
nit: the casing on this is difficult to read. Could it be camelCase like `ewmaAlpha`?